druid/src/env.rs

```diff
@@ -240,6 +240,19 @@ impl Env {
     /// [`WidgetExt::debug_widget`]: trait.WidgetExt.html#method.debug_widget
     pub const DEBUG_WIDGET: Key<bool> = Key::new("org.linebender.druid.built-in.debug-widget");
 
+    /// Creates a new empty Env with localization information.
+    pub fn new() -> Self {
+        let l10n = L10nManager::new(vec!["builtin.ftl".into()], "./resources/i18n/");
```

Comment:
huh, I hadn't really considered this. We really don't want to be re-creating the `L10Manager` everytime we create an env, if we're creating them regularly to just use as a key/value store; we'll need a different approach to that.

Reply:
Ok, I wasn't sure how to go about that. I added this so someone can create an Env when they use it statically.

Reply:
is there a better way to come up with empty Env? I think instead of the static variant using a Env, it should take a closure like this
| data: &T, env: &mut Env | {}
This way the user doesn't concern themselves with creating an env and instead should modify the one provided. We won't have to worry about creating an env without localization resources or with it. The env provided to the closure would be a clone of the env provided by the widget method.